vm/syncervm_client.go

```diff
@@ -268,72 +263,3 @@ func (s *stateSyncerClient) UpdateSyncTarget(b *chain.StatelessBlock) (bool, err
 	s.targetUpdated = true // Set [targetUpdated] so we call SetLastAccepted on finish
 	return true, nil       // Sync root target updated successfully
 }
-
-// startingSync is called before [AcceptedSyncableBlock] returns
-func (s *stateSyncerClient) startingSync(state bool) {
-	s.startedSync = true
-	vm := s.vm
-
-	// If state sync, we pessimistically assume nothing we have on-disk will
-	// be useful (as we will jump ahead to some future block).
-	if state {
-		return
-	}
-
-	// Exit early if we don't have any blocks other than genesis (which
```

Comment:
Turns out this backfill was not being triggered in a number of cases where it should have. I moved to `vm` to trigger during ANY bootstrap.